.github/workflows/main.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Tests
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    env:
+      SECRET_KEY: foobarbaz
+      GOOGLE_MAPS_API_KEY: $GOOGLE_MAPS_API_KEY
+
+    services:
+      postgres:
+        image: postgres:10.8
+        env:
+          POSTGRES_USER: test
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: systersdb
+        ports:
+        - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements/dev.txt
+        sudo apt-get install python-gdal
+    
+    - name: Run Flake Test
```

Comment:
@satya7289 Could we have the flake tests after the unit tests. Just a suggestion

Reply:
I intensionally do that because I am thinking contributor first resolve flake error and then unit test. Let's know others thought on this.

Reply:
Sounds good to me! Fail fast!